scavenger-api/src/main/resources/application-local.yml

```diff
@@ -3,7 +3,7 @@ scavenger:
 
 spring:
   datasource:
-    url: jdbc:h2:mem:scavenger;MODE=MySQL;DB_CLOSE_DELAY=-1;CASE_INSENSITIVE_IDENTIFIERS=TRUE
+    url: jdbc:h2:tcp://localhost:9092/mem:scavenger;MODE=MySQL;DB_CLOSE_DELAY=-1;CASE_INSENSITIVE_IDENTIFIERS=TRUE
```

Comment:
This change requires that when running the scavenger-api with a local profile, the collector is run first.
The local profile is to make it simple for scavneger developers or contributors to run their modules to develop features without dependencies on other modules.

Reply:
> This change requires that when running the scavenger-api with a local profile, the collector is run first.

That's correct

> The local profile is to make it simple for scavneger developers or contributors to run their modules to develop features without dependencies on other modules.

Ah that makes more sense. Right now it isn't clear in the documentation (`installation.md`) that the `local` profile cannot actually be used to demo agent collection in the web UI.

In that case, I can decline this PR and propose some documentation updates instead which makes it clear that the Collector and Web UI cannot communicate with each other in the `local` profile?

Reply:
Yes, that's good. Can you contribute from another PR?